backend/src/main/java/codezap/global/pagination/FixedPageCounter.java

```diff
@@ -0,0 +1,32 @@
+package codezap.global.pagination;
+
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Component;
+
+import com.querydsl.core.types.dsl.BooleanExpression;
+import com.querydsl.core.types.dsl.EntityPathBase;
+import com.querydsl.jpa.impl.JPAQueryFactory;
+
+@Component
+public class FixedPageCounter {
+
+    private static final int MAXIMUM_PAGE = 5;
+
+    public int countNextFixedPage(
+            JPAQueryFactory queryFactory,
+            EntityPathBase<?> entityPath,
+            Pageable pageable,
+            BooleanExpression... conditions
+    ) {
+        int maximumElementsCount = pageable.getPageSize() * MAXIMUM_PAGE;
+        long nextFixedElementCounts = queryFactory
+                .selectFrom(entityPath)
+                .where(conditions)
+                .offset(pageable.getOffset())
+                .limit(maximumElementsCount)
+                .fetch()
+                .size();
```

Comment:
```suggestion
        long nextFixedElementCounts = queryFactory.selectFrom(entityPath)
                .where(conditions)
                .offset(pageable.getOffset())
                .limit(maximumElementsCount)
                .fetch()
                .size();
```

아주아주 사소하지만,,, stream() 메서드를 붙여서 쓰는 것 처럼 `selectFrom()` 도 동일한 형식으로 포매팅 해 주면 더 읽기 편할 것 같아요!